crates/compiler/builtins/roc/Num.roc

```diff
@@ -867,6 +868,41 @@ max = \a, b ->
     else
         b
 
+## Restricts a number to be between two other numbers.
+##
+## ```
+## restrictToInterval 5 { startAt: 3, endAt: 8 } == 5
+## restrictToInterval 1 { startAt: 3, endAt: 8 } == 3
+## restrictToInterval 10 { startAt: 3, endAt: 8 } == 8
+
+## restrictToInterval 5 { startAt: 8, endAt: 3 } == 5
+## restrictToInterval 1 { startAt: 8, endAt: 3 } == 3
+## restrictToInterval 10 { startAt: 8, endAt: 3 } == 8
+## ```
+## You may know this function as `clamp` in other languages.
```

Comment:
I'd remove this from the Roc documentation. On the docs website, there is a way to list functions with other names in other languages: https://github.com/roc-lang/roc/blob/26630ee3af135abaaaa39fc06d321dff42bb02ad/www/public/different-names/index.html#L19

Reply:
Oh, I see you've already done the later.